src/routes/api/accommodations.js

```diff
@@ -205,6 +202,6 @@ router.patch('/book', validateToken, checkBookingRole, validateBooking, bookAcco
 router.get('/:slug', validateToken, viewSpecificAccommodation);
 router.patch('/activate/:slug', validateToken, validateReasons, accommodationActivation);
 router.get('/admin/deactivated', validateToken, viewDeactivated);
-
+router.post('/:id/like', validateToken, checkId, likeValidation, likeAccommodation);
```

Comment:
Make this route dynamic to cater for like and dislike you can add something like this `/:id/:action`